subsys/bluetooth/mesh/brg_cfg_cli.c

```diff
@@ -5,12 +5,158 @@
+		cli->cb->subnet_bridge_status(cli, ctx->addr, status);
+	}
+	return 0;
+}
+
+static int bridging_table_status(const struct bt_mesh_model *model, struct bt_mesh_msg_ctx *ctx,
+				struct net_buf_simple *buf)
+{
+	struct bt_mesh_bridging_table_status table_status;
+	struct bt_mesh_bridging_table_status *rsp;
+	uint32_t net_key_indices;
+
+	table_status.status = net_buf_simple_pull_u8(buf);
+	table_status.entry.directions = net_buf_simple_pull_u8(buf);
+	net_key_indices = net_buf_simple_pull_le24(buf);
+	table_status.entry.net_idx1 = net_key_indices & BIT_MASK(12);
+	table_status.entry.net_idx2 = (net_key_indices >> 12) & BIT_MASK(12);
+	table_status.entry.addr1 = net_buf_simple_pull_le16(buf);
+	table_status.entry.addr2 = net_buf_simple_pull_le16(buf);
+
+	if (!(table_status.entry.addr1 == BT_MESH_ADDR_UNASSIGNED ||
+	     BT_MESH_ADDR_IS_UNICAST(table_status.entry.addr1))) {
+		LOG_ERR("addr1 shall be a unicast address or unassigned.");
+		return -EINVAL;
+	} else if (table_status.entry.addr2 == BT_MESH_ADDR_ALL_NODES) {
+		LOG_ERR("addr2 shall not be the all-nodes fixed group address.");
+		return -EINVAL;
+	}
+
+	if (bt_mesh_msg_ack_ctx_match(&cli->ack_ctx, OP_BRIDGING_TABLE_STATUS, ctx->addr,
+				      (void **)&rsp)) {
+		*rsp = table_status;
+		bt_mesh_msg_ack_ctx_rx(&cli->ack_ctx);
+	}
+
+	if (cli->cb && cli->cb->bridging_table_status) {
+		cli->cb->bridging_table_status(cli, ctx->addr, &table_status);
+	}
+	return 0;
+}
+
+static int bridged_subnets_list(const struct bt_mesh_model *model, struct bt_mesh_msg_ctx *ctx,
+				struct net_buf_simple *buf)
+{
+	struct bt_mesh_bridged_subnets_list subnets_list;
+	struct bt_mesh_bridged_subnets_list *rsp;
+	uint16_t net_idx_filter;
+
+	net_idx_filter = net_buf_simple_pull_le16(buf);
+	subnets_list.net_idx_filter.filter = net_idx_filter & BIT_MASK(2);
+	subnets_list.net_idx_filter.net_idx = (net_idx_filter >> 4) & BIT_MASK(12);
+	subnets_list.start_idx = net_buf_simple_pull_u8(buf);
+
+	if (buf->len) {
+		subnets_list.list = buf;
+	} else {
+		subnets_list.list = NULL;
+	}
+
+	if (bt_mesh_msg_ack_ctx_match(&cli->ack_ctx, OP_BRIDGED_SUBNETS_LIST, ctx->addr,
+				      (void **)&rsp)) {
+		*rsp = subnets_list;
+		bt_mesh_msg_ack_ctx_rx(&cli->ack_ctx);
+	}
+
+	if (cli->cb && cli->cb->bridged_subnets_list) {
+		cli->cb->bridged_subnets_list(cli, ctx->addr, &subnets_list);
+	}
+	return 0;
+}
+
+static int bridging_table_list(const struct bt_mesh_model *model, struct bt_mesh_msg_ctx *ctx,
+			      struct net_buf_simple *buf)
+{
+	struct bt_mesh_bridging_table_list table_list;
+	struct bt_mesh_bridging_table_list *rsp;
+	uint32_t net_key_indices;
+
+	table_list.status = net_buf_simple_pull_u8(buf);
+	net_key_indices = net_buf_simple_pull_le24(buf);
+	table_list.net_idx1 = net_key_indices & BIT_MASK(12);
+	table_list.net_idx2 = (net_key_indices >> 12) & BIT_MASK(12);
+	table_list.start_idx = net_buf_simple_pull_le16(buf);
+
+	if ((table_list.status == STATUS_SUCCESS) && buf->len) {
```

Comment:
`buf->len` should be in multiples of `4` octets.

Reply:
Multiples of 5, no? 
![image](https://github.com/user-attachments/assets/40f0b606-8691-4f8b-9783-36ab9a386e47)